packages/core/editor/src/parsers/deserializeListNodes.ts

```diff
@@ -0,0 +1,50 @@
+import {Descendant} from 'slate';
```

Comment:
So, I'll explain first how List blocks works.
By design, each `LI` element is essentially separate block. And each block contains only one Slate element: 
-  Block `TodoList` contains only the `todo-list`
- Block `NumberedList` contains only the `numbered-list` 
- Block `BulletedList` contains only the `bulleted-list` 

What does it mean? 
This means that our general function should return non-nested Slate elements, it should return an array of blocks with the correct `depth` in the `meta' field of the block, because in our case it is the `depth` field that will determine the nesting of blocks